schema/gkrule-source.yaml

```diff
@@ -0,0 +1,144 @@
+$schema: "http://json-schema.org/draft/2020-12/schema"
+title: "GA4GH-Genomic-Knowledge-Rule-Definitions"
```

Comment:
change to `CategoricalVariationRule` ? (sorry I know you want to strangle me at this point).  I just think that if we are confident that these rules (or categorical rules) are always returning Variation based on categorically defined rules using genomic attributes and characteristics then we should not make this so general. And as I mentioned in the other comment(s) "Knowledge" is not really in play here.  I get that this is a play off of the GKS group name, but it may not really be technically appropriate to embed in the name of the super class.